pytorch_lightning/core/lightning.py

```diff
@@ -990,6 +990,30 @@ def predict(self, batch: Any, batch_idx: int, dataloader_idx: Optional[int] = No
         """
         return self(batch)
 
+    def configure_callbacks(self):
```

Comment:
add a type-hint that this returns a list of callbacks?

Reply:
William once said he doesn't want type-hints in hooks api that  users override, the reason being that IDE's autocomplete it when overriding the hooks from the superclass. Then users would manually have to delete it. He sees this as not user friendly. So I'm not sure if I should add it :)